contracts/utils/Address.sol

```diff
@@ -76,26 +80,50 @@ library Address {
         if (address(this).balance < value) {
             revert Errors.InsufficientBalance(address(this).balance, value);
         }
-        (bool success, bytes memory returndata) = target.call{value: value}(data);
-        return verifyCallResultFromTarget(target, success, returndata);
```

Comment:
I think `verifyCallResultFromTarget(..)` is not used internally anymore (and not tested).
We can eventually in 5.x add unit tests for it (and deprecate usage until 6.0 removal if it is considered to be useless for developer, which am not sure about)